lib/zendesk_apps_support/package.rb

```diff
@@ -212,6 +215,16 @@ def has_requirements?
       has_file?(REQUIREMENTS_FILENAME)
     end
 
+    def self.has_custom_object_requirements?(requirements_hash)
```

Comment:
This isn't great. I want to return false when requirements_hash is any of these values: `nil`, `{}`, `{ custom_object_types: {}, custom_object_relationship_types: {} }`. We're using `present?` in the previous version which isn't available here. Keen to hear if anyone has a nicer way to do this.